packages/cli/create-strapi-app/src/create-strapi-app.ts

```diff
@@ -61,8 +56,7 @@ async function generateApp(
   }
 
   if (!options.skipCloud) {
-    checkRequirements();
-    await handleCloudProject(projectName);
+    await handleCloudProject();
```

Comment:
```suggestion
    await handleCloudLogin();
```

Probably better to rename the function to something more representative